docs/developers/api-client.md

```diff
@@ -565,6 +565,30 @@ Both will give you something like:
  'total_number_of_events_sent_by_client': 1}
 ```
 
+### Add Attributes to Events
+
+To add new attributes to event an event:
+
+```python
+# event_id and searchindex_id can be found by searching for the target event.
+
+event = [
+  {
+    '_id': event_id,
+    '_index': searchindex_id,
+    '_type': '_doc',
+    'attributes': [
+      {
+        'attr_name': 'A new attribute name',
```

Comment:
I wonder if we should encourage people to use whitespaces in attribute names.... WDYT?